src/Link.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { Text } from 'react-native';
+import PropTypes from 'prop-types';
+
+import normalize from './helpers/normalize';
+import GalioTheme, { withGalio } from './theme';
+
+function Typography({
+    style,
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6,
+    p,
+    body,
+    small,
+    muted,
+    neutral,
+    size,
+    color,
+    bold,
+    italic,
+    center,
+    children,
+    styles,
+    theme,
+    ...rest
+  }) {
+    return (
+        <TouchableOpacity onPress={() => Linking.openURL(url)}>
+            <Text style={[
+                h1 && { fontSize: normalize(44) },
+                h2 && { fontSize: normalize(38) },
+                h3 && { fontSize: normalize(30) },
+                h4 && { fontSize: normalize(24) },
+                h5 && { fontSize: normalize(21) },
+                h6 && { fontSize: normalize(18) },
+                p && { fontSize: normalize(16) },
+                body && { fontSize: normalize(14) },
+                small && { fontSize: normalize(12) },
+                muted && { color: theme.COLORS.MUTED },
+                neutral && { color: theme.COLORS.NEUTRAL },
+                size && { fontSize: size },
+                color && { color },
+                italic && { fontStyle: 'italic' },
+                bold && { fontWeight: 'bold' },
+                center && { textAlign: 'center' },
+                style && style,
+                ]}
+                {...rest}>
+                {children}>
+                Google
+            </Text>
+        </TouchableOpacity>
+    );
+  }
+  
```

Comment:
You need to add the default styles of the Link component. The default color is Primary.